misc/python/materialize/rustc_flags.py

```diff
@@ -21,3 +23,76 @@
     "-Cinstrument-coverage",
     "-Cllvm-args=-runtime-counter-relocation",
 ]
+
+
+class Sanitizer(Enum):
+    """What sanitizer to use"""
+
+    address = "address"
+    """The AddressSanitizer, see https://clang.llvm.org/docs/AddressSanitizer.html"""
+
+    hwaddress = "hwaddress"
+    """The HWAddressSanitizer, see https://clang.llvm.org/docs/HardwareAssistedAddressSanitizerDesign.html"""
+
+    cfi = "cfi"
+    """Control Flow Integrity, see https://clang.llvm.org/docs/ControlFlowIntegrity.html"""
+
+    thread = "thread"
+    """The ThreadSanitizer, see https://clang.llvm.org/docs/ThreadSanitizer.html"""
+
+    leak = "leak"
+    """The LeakSanitizer, see https://clang.llvm.org/docs/LeakSanitizer.html"""
+
+    undefined = "undefined"
```

Comment:
Shall we call it `undefined_behavior`? (optional)

Reply:
The compiler flag is also `-fsanitize=undefined`, so I'd prefer to stay with the current one.